### DISCUSSION TITLE
Yagiz Nizipli

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#235] [Announcements]
- https://twitter.com/yagiznizipli
- https://www.yagiz.co/

## Topics

- Node
- Open Source
- ADA Url Parser
- Node perf

## Questions

- [x] What your involvement with nodejs?
- [x] What is the ADA URL parser and how did you make it so fast? 
- [x] What were some challenges building it? How different can URLs be?
- [x] How does work get decided on for nodejs? is it all volunteer? is all your work volunteer?
- [x] Before we get to perf. what other features have you championed in node?

### Perf

- [x] What led you to founding the node perf team? How does it steer the work done on node?
- [x] What work have you done to make node.js faster?
- [x] What do you think about the new runtimes popping up? Do you think they should have collaborated instead?
- [x] While I'm sure you work on what you can, there must be some other large perf constraints in the project. Whats the biggest one you see that if tackled could be huge?

### Future

- [x] What are your thoughts on the state of funding in OSS?
- [x] What do you think the future of javascript backend runtimes is?

**Comment by anonrig** [comment]
- What is Pacquet (https://github.com/pnpm/pacquet) and what’s your involvement?

**Comment by vramana** [comment]
If I recall correctly, undici was faster than the previous C/C++ implementation. Do you know why? Will we see a port back to native code? Is there any other submodule you want to make it blazingly fast?

**Comment by hipstersmoothie** [comment]
Tooltips: 

https://zackoverflow.dev/writing/flappy-bird-in-type-level-typescript
https://github.com/farm-fe/farm

**Comment by anonrig** [comment]
Tooltips:

https://ui.com/us/camera-security
https://prsguitars.com/electrics/model/silver_sky_2021

**Comment by just-be-dev** [comment]
- https://github.com/seanoliver/audioflare
- https://wasmer.io/posts/announcing-winterjs-service-workers
